src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/DiagnosticsExtensionTests.ps1

```diff
@@ -237,7 +237,7 @@ function Test-VmssDiagnosticsExtension
         $vmssType = 'Microsoft.Compute/virtualMachineScaleSets';
 
         $adminUsername = 'Foo12';
-        $adminPassword = "BaR@123" + $rgname;
+        $adminPassword = $PLACEHOLDER;
```

Comment:
instead of a place holder, you shoulduse [this function](https://github.com/Azure/azure-powershell/pull/7190/files#diff-c3a91e9dd099c92910feae5283a65883R28) to get a password.